main.js

```diff
@@ -0,0 +1,7 @@
+var logger
+
+if (typeof global !== 'undefined' && global.console) {
```

Comment:
it's a valid detection of a global console which is good enough to keep nashorn integration functional. I don't see a reason to be more particular for the time being.

any preferences?